kong/plugins/rate-limiting/policies/init.lua

```diff
@@ -167,7 +167,12 @@ return {
         current_metric = nil
       end
 
-      return current_metric and current_metric or 0
+      local ok, err = red:set_keepalive(10000, 100)
```

Comment:
If that is the case, possibly, but better simply not include new occurences in the first place.